tests/test_csc.py

```diff
@@ -815,6 +817,91 @@ async def test_show_alarms(self):
                 assert data.acknowledgedBy == ""
             assert seen_alarm_names == alarm_names
 
+    @patch("aiohttp.ClientSession.post")
+    async def test_make_log_entry(self, mock_post):
+        """Test the makeLogEntry command."""
+
+        def dict_subset(smallDict, largeDict):
+            """Return subset of dict using only keys found in smallDict."""
+            return {k: v for k, v in largeDict if k in smallDict}
+
+        # Send the makeLogEntry command.
+        async with self.make_csc(
+            config_dir=TEST_CONFIG_DIR, initial_state=salobj.State.STANDBY
+        ), salobj.Controller(name="ATDome", write_only=True) as atdome:
+            atdome_alarm_name = "Enabled.ATDome:0"
+
+            await salobj.set_summary_state(
+                self.remote, state=salobj.State.ENABLED, override="enabled.yaml"
+            )
+            await self.check_all_alarms_events_are_none()
+
+            await atdome.evt_summaryState.set_write(
+                summaryState=salobj.State.FAULT, force_output=True
+            )
+            await self.assert_next_alarm(
+                name=atdome_alarm_name,
+                severity=AlarmSeverity.CRITICAL,
+                maxSeverity=AlarmSeverity.CRITICAL,
+                acknowledged=False,
+                acknowledgedBy="",
+            )
+
+            test_case_json = {"key": "TEST-123", "status": "PASS"}
+            mock_response = AsyncMock()
+            mock_response.json.return_value = test_case_json
+            mock_post.return_value.__aenter__.return_value = mock_response
+
+            test_response = await self.remote.cmd_makeLogEntry.set_start(
+                name=atdome_alarm_name,
+                timeout=STD_TIMEOUT,
+            )
+
+            # The actual test_response will have more fields than expected.
+            # We don't care about most of them and they might cause variations
+            # in results so construct a subset of the the actual for only the
+            # keys in expected.
+            expected_response = {
```

Comment:
You can remove this test here of the `test_reponse`.

Reply:
Why? If my cmd_makeLogEntry was broken in a certain way, wouldn't that response be different?

Reply:
no.. if your command was broken it would raise an exception

Reply:
I guess you are saying that there is no possibility of it returning a different value of any keys in the response.  

Maybe we would want underlying command to, for example, notice that the narrativelog API returned something different than the posted payload (its supposed to return the full record that was inserted) and indicate so with a different response.

Reply:
>I guess you are saying that there is no possibility of it returning a different value of any keys in the response.

yes